packages/extensions/openapi-to-cadl/test/anomalyDetector/cadl-output/models.tsp

```diff
@@ -1,81 +1,84 @@
-import "@cadl-lang/rest";
-import "@azure-tools/cadl-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-core";
 
-using Cadl.Rest;
+using TypeSpec.Rest;
+using TypeSpec.Http;
 using Azure.Core;
 
 namespace AnomalyDetectorClient;
 
 enum ApiVersion {
-  "v1.1-preview.1",
+  `v1.1-preview.1`,
```

Comment:
Why did these quotes change?

Reply:
This seems to come from a recent change in the prettier extension https://github.com/microsoft/typespec/commit/766cdb728f662c8fdde6417854a301d615ed7cfc#diff-68104c1f8d7943b8b54125fab0d85436006884c0d27ec0cda487d209f9345835